### PR TITLE
0.10.2 release prep

### PR DESCRIPTION
**Pull request by bmcfee**
#### What does this implement/fix? Explain your changes.

This PR updates documentation and version data for the 0.10.2 release.

## ToDos

- [x] Finalize release date
- [ ] Update DOI index

### CODE REVIEW

**Comment by alumkal** [comment]
`Jize Guo <https://github.com/alumkal>`

File: AUTHORS.md [hunk 1]
```diff
@@ -104,6 +104,10 @@ Contributors
 * Shin Hyun <https://github.com/kyaryunha>
 * Iliya S <https://github.com/zenitismus>
 * Eugene Rabinovich <https://github.com/erabinov>
+* Gao Lei <https://github.com/alvingao>
+* alumkal <https://github.com/alumkal>
```
Comment (bmcfee) [inline AUTHORS.md:108]:
```suggestion
* Jize Guo <https://github.com/alumkal>
```

File: docs/changelog.rst [hunk 2]
```diff
@@ -5,6 +5,68 @@ Changelog
...
+    - `#1821`_ Added "copy button" to code examples in documentation. *Brian McFee*
+    - `#1789`_ Fixed type annotations for sampling rate parameters. *Brian McFee*
+    - `#1780`_ Corrected mistake in `util.frame` documentation. *Gao Lei*
+    - `#1783`_ Removed problematic language from test code and documentation. *Dan Ellis*
+
+Other changes and maintenance
+    - `#1826`_ Expanded parameter pass-through support in `effects.hpss` and related functions. *Brian McFee*
+    - `#1822`_ Allow `display.specshow` to control min and max tempo in tempogram plots. *Brian McFee*
+    - `#1820`_ Updated github actions. *Brian McFee*
+    - `#1813`_ Improved performance of `autocorrelate`. *@alumkal*
```
Comment (bmcfee) [inline docs/changelog.rst:37]:
```suggestion
    - `#1813`_ Improved performance of `autocorrelate`. *Jize Guo*
```

**Comment by psmskelton** [comment]
@bmcfee Thanks. Phillip S.M. Skelton <https://github.com/psmskelton/>

File: AUTHORS.md [hunk 3]
```diff
@@ -104,6 +104,10 @@ Contributors
 * Shin Hyun <https://github.com/kyaryunha>
 * Iliya S <https://github.com/zenitismus>
 * Eugene Rabinovich <https://github.com/erabinov>
+* Gao Lei <https://github.com/alvingao>
+* Jize Guo <https://github.com/alumkal>
+* psmskelton <https://github.com/psmskelton>
```
Comment (bmcfee) [inline AUTHORS.md:109]:
```suggestion
* Phillip S.M. Skelton <https://github.com/psmskelton>
```

**Pull request merged**